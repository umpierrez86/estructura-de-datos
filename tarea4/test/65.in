# implementar pila
cantidadEnPila
apilar (3,3.0)
cantidadEnPila
cima
desapilar
cantidadEnPila
apilar (5,5.0)
apilar (6,5.0)
apilar (3,5.0)
apilar (8,5.0)
apilar (2,5.0)
apilar (1,5.0)
apilar (10,5.0)
apilar (5,3.0)
cima
cantidadEnPila
desapilar
cima
cantidadEnPila
apilar (21,5.0)
apilar (18,5.0)
apilar (13,5.0)
apilar (30,5.0)
apilar (1,5.0)
apilar (1,5.0)
cima
cantidadEnPila
Fin