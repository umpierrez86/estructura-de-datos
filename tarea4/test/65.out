1>#  implementar pila.
2>Cantidad en pila: 0.
3>Apilando (3,3.00)
4>Cantidad en pila: 1.
5>(3,3.00)
6>Desapilando.
7>Cantidad en pila: 0.
8>Apilando (5,5.00)
9>Apilando (6,5.00)
10>Apilando (3,5.00)
11>Apilando (8,5.00)
12>Apilando (2,5.00)
13>Apilando (1,5.00)
14>Apilando (10,5.00)
15>Apilando (5,3.00)
16>(5,3.00)
17>Cantidad en pila: 8.
18>Desapilando.
19>(10,5.00)
20>Cantidad en pila: 7.
21>Apilando (21,5.00)
22>Apilando (18,5.00)
23>Apilando (13,5.00)
24>Apilando (30,5.00)
25>Apilando (1,5.00)
26>Apilando (1,5.00)
27>(1,5.00)
28>Cantidad en pila: 13.
29>Fin.
